patch/0001-armhf-mandatory-configs-for-boot.patch

```diff
@@ -0,0 +1,201 @@
+ CONFIG_PERF_EVENTS=y
+ # CONFIG_DEBUG_PERF_USE_VMALLOC is not set
+ CONFIG_VM_EVENT_COUNTERS=y
+-# CONFIG_COMPAT_BRK is not set
++CONFIG_COMPAT_BRK=y
+ CONFIG_SLAB=y
+ # CONFIG_SLUB is not set
+ # CONFIG_SLOB is not set
+@@ -1736,7 +1736,7 @@ CONFIG_TEGRA_AHB=y
+ #
+ # CONFIG_UEVENT_HELPER is not set
+ CONFIG_DEVTMPFS=y
+-# CONFIG_DEVTMPFS_MOUNT is not set
++CONFIG_DEVTMPFS_MOUNT=y
+ CONFIG_STANDALONE=y
+ CONFIG_PREVENT_FIRMWARE_BUILD=y
+ CONFIG_FW_LOADER=y
+@@ -1793,16 +1793,16 @@ CONFIG_PROC_EVENTS=y
+ CONFIG_MTD=y
+ # CONFIG_MTD_TESTS is not set
+ # CONFIG_MTD_REDBOOT_PARTS is not set
+-# CONFIG_MTD_CMDLINE_PARTS is not set
++CONFIG_MTD_CMDLINE_PARTS=y
+ # CONFIG_MTD_AFS_PARTS is not set
+-CONFIG_MTD_OF_PARTS=m
+-CONFIG_MTD_AR7_PARTS=m
++CONFIG_MTD_OF_PARTS=y
++CONFIG_MTD_AR7_PARTS=y
+ 
+ #
+ # User Modules And Translation Layers
+ #
+-CONFIG_MTD_BLKDEVS=m
+-CONFIG_MTD_BLOCK=m
++CONFIG_MTD_BLKDEVS=y
++CONFIG_MTD_BLOCK=y
+ CONFIG_MTD_BLOCK_RO=m
+ # CONFIG_FTL is not set
+ # CONFIG_NFTL is not set
+@@ -1817,8 +1817,10 @@ CONFIG_MTD_SWAP=m
+ #
+ # RAM/ROM/Flash chip drivers
+ #
+-# CONFIG_MTD_CFI is not set
++CONFIG_MTD_CFI=y
+ # CONFIG_MTD_JEDECPROBE is not set
++CONFIG_MTD_GEN_PROBE=y
++# CONFIG_MTD_CFI_ADV_OPTIONS is not set
+ CONFIG_MTD_MAP_BANK_WIDTH_1=y
+ CONFIG_MTD_MAP_BANK_WIDTH_2=y
+ CONFIG_MTD_MAP_BANK_WIDTH_4=y
+@@ -1829,6 +1831,10 @@ CONFIG_MTD_CFI_I1=y
+ CONFIG_MTD_CFI_I2=y
+ # CONFIG_MTD_CFI_I4 is not set
+ # CONFIG_MTD_CFI_I8 is not set
++CONFIG_MTD_CFI_INTELEXT=y
++CONFIG_MTD_CFI_AMDSTD=y
++CONFIG_MTD_CFI_STAA=y
++CONFIG_MTD_CFI_UTIL=y
+ CONFIG_MTD_RAM=m
+ # CONFIG_MTD_ROM is not set
+ # CONFIG_MTD_ABSENT is not set
+@@ -1850,8 +1856,8 @@ CONFIG_MTD_PLATRAM=m
+ CONFIG_MTD_DATAFLASH=m
+ # CONFIG_MTD_DATAFLASH_WRITE_VERIFY is not set
+ # CONFIG_MTD_DATAFLASH_OTP is not set
+-CONFIG_MTD_M25P80=m
+-CONFIG_MTD_SST25L=m
++CONFIG_MTD_M25P80=y
++CONFIG_MTD_SST25L=y
+ # CONFIG_MTD_SLRAM is not set
+ # CONFIG_MTD_PHRAM is not set
+ # CONFIG_MTD_MTDRAM is not set
+@@ -1878,7 +1884,7 @@ CONFIG_MTD_NAND_RICOH=m
+ # CONFIG_MTD_NAND_DISKONCHIP is not set
+ # CONFIG_MTD_NAND_DOCG4 is not set
+ CONFIG_MTD_NAND_CAFE=m
+-CONFIG_MTD_NAND_PXA3xx=m
++CONFIG_MTD_NAND_PXA3xx=y
+ CONFIG_MTD_NAND_NANDSIM=m
+ CONFIG_MTD_NAND_GPMI_NAND=m
+ # CONFIG_MTD_NAND_BRCMNAND is not set
+@@ -1902,12 +1908,12 @@ CONFIG_MTD_ONENAND_2X_PROGRAM=y
+ CONFIG_MTD_LPDDR=m
+ CONFIG_MTD_QINFO_PROBE=m
+ # CONFIG_MTD_LPDDR2_NVM is not set
+-CONFIG_MTD_SPI_NOR=m
++CONFIG_MTD_SPI_NOR=y
+ # CONFIG_MTD_MT81xx_NOR is not set
+ CONFIG_MTD_SPI_NOR_USE_4K_SECTORS=y
+ # CONFIG_SPI_CADENCE_QUADSPI is not set
+ # CONFIG_SPI_FSL_QUADSPI is not set
+-CONFIG_MTD_UBI=m
++CONFIG_MTD_UBI=y
+ CONFIG_MTD_UBI_WL_THRESHOLD=4096
+ CONFIG_MTD_UBI_BEB_LIMIT=20
+ # CONFIG_MTD_UBI_FASTMAP is not set
+@@ -3266,13 +3272,14 @@ CONFIG_VT_CONSOLE_SLEEP=y
+ CONFIG_HW_CONSOLE=y
+ CONFIG_VT_HW_CONSOLE_BINDING=y
+ CONFIG_UNIX98_PTYS=y
+-# CONFIG_LEGACY_PTYS is not set
++CONFIG_LEGACY_PTYS=y
```

Comment:
why do we need this?

Reply:
I will check this and update if not needed.

Reply:
Removed the change as it is not needed